terrawrap/utils/config.py

```diff
@@ -86,6 +89,152 @@ def parse_wrapper_configs(wrapper_config_files: List[str]) -> WrapperConfig:
     return wrapper_config_obj
 
 
+def is_config_directory(directory: str) -> bool:
+    """
+    Checks if a wrapper file directory is a config_directory
+    :param directory: A wrapper files directory
+    :return: A boolean True if the config section of the wrapper file is not False or doesn't exist.
+    """
+    config = False
+    for file in os.listdir(directory):
+        if file.endswith(".tf"):
+            config = True
+    return config
+
+
+def has_depends_on(directory: str) -> bool:
```

Comment:
given a directory containing a wrapper file, it checks if the wrapper file has a depends_on field. If so, this means that it is set up to be added to the dependency graph.